documentation/general/publish.md

```diff
@@ -51,8 +51,8 @@ the last 90 days
 _Note:
 [GitHub's documentation provides screenshots for the various steps](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch)_
 
-1. Navigate to the `openverse-api` "Actions" tab in GitHub:
-   [github.com/WordPress/openverse-api/actions](https://github.com/WordPress/openverse-api/actions).
+1. Navigate to the `openverse` "Actions" tab in GitHub:
+   [github.com/WordPress/openverse/actions](https://github.com/WordPress/openverse/actions).
```

Comment:
This is such an interesting workflow, I was surprised I hadn't heard of it or come across it in the code. Funnily it has never been executed once.

<img width="1317" alt="image" src="https://github.com/WordPress/openverse/assets/16580576/b48fb716-20b2-4890-91a2-cf90606602a3">

Reply:
Looks like a good one to remove!